src/commands/sentinel-pending-scripts.json

```diff
@@ -10,6 +10,16 @@
             "ADMIN",
             "SENTINEL",
             "ONLY_SENTINEL"
-        ]
+        ],
+        "reply_schema": {
+            "type": "array",
+            "description": "List of pending scripts.",
+            "items": {
```

Comment:
i think that like `function|list` we wanna describe the fields (argv, flags, pid, run-time, run-delay, retry-num).

Reply:
yes, updated, please take a look.